vendor/github.com/seccomp/libseccomp-golang/BUILD

```diff
@@ -7,12 +7,6 @@ go_library(
         "seccomp_internal.go",
     ],
     cgo = True,
-    clinkopts = select({
```

Comment:
I think this might actually be a problem (see also review comment below, `seccomp.go` line 23)

https://github.com/bazelbuild/bazel-gazelle/blob/44ce230b3399a5d4472198740358fcd825b0c3c9/language/go/fileinfo.go#L388

https://github.com/bazelbuild/bazel-gazelle/issues/203